_includes/releases/babelfish-1.0.1.md

```diff
@@ -0,0 +1,13 @@
+
+- Babelfish Version: 1.0.1
+- PostgreSQL Server Version: 13.4
+- Download Source distributions:
+  - [BABEL_1_0_1__PG_13_4.zip](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_1_0_1__PG_13_4/BABEL_1_0_1__PG_13_4.zip)
+  - [BABEL_1_0_1__PG_13_4.tar.gz](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_1_0_1__PG_13_4/BABEL_1_0_1__PG_13_4.tar.gz)
+- Minimum Compass Version: [1.0.0](https://github.com/babelfish-for-postgresql/babelfish_compass/releases/tag/v1.0)
+- Date: Nov 8, 2022
+
+
```

Comment:
We should additionally provide the output of below command for checking if Babelfish upgrade was successful:
```bash
babelfish_db[124386]# SELECT sys.serverProperty('BabelfishVersion') AS BabelfishVersion;
 babelfishversion 
------------------
 1.0.1
(1 row)
```

Reply:
Updated...